spec2nii/GE/ge_pfile.py

```diff
@@ -271,6 +273,143 @@ def _process_gaba(pfile):
+          ( 1) Short TE Reference : 35ms Unsupressed Water
+          (32) Short TE Unedited  : 35ms Water Suppressed PRESS
+
+        Data is directly separated from the raw data (pfile.map.raw_data) where the data
+          mapper (GABA mapper) is simply used to populate in the raw data.
+
+        Author : Aaron Gudmundson, Johns Hopkins University, 2024
+        Contact: agudmun2@jhmi.edu
+    """
+
+    # Additional Imports
+    import copy
+
+    # Editing Parameters
+    edit_cases       = 4                                                                    # 4 Editing Conditions
+    edit_pulse_1     = 4.58                                                                 # 4.58 ppm
+    edit_pulse_2     = 1.90                                                                 # 1.90 ppm
+    edit_pulse_4     = 4.18                                                                 # 4.18 ppm
+    pulse_length     = 0.02                                                                 # Edit Pulse 20 ms
+
+    dim_header       = {'EditCondition': ['A', 'B', 'C', 'D']}                              # 4 Subscans
+    edit_pulse_val   = {'A': {'PulseOffset': [edit_pulse_1, edit_pulse_2], 'PulseDuration': pulse_length},
+                        'B': {'PulseOffset': [edit_pulse_4, edit_pulse_2], 'PulseDuration': pulse_length},
+                        'C': {'PulseOffset': edit_pulse_1, 'PulseDuration': pulse_length},
+                        'D': {'PulseOffset': edit_pulse_4, 'PulseDuration': pulse_length}}
+
+    # All Data (Skip 1st Transient - GE automatically has historically included a 'noise' transient)
+    raw_data         = pfile.map.raw_data[:, :, :, :, 1:, :]                                # Raw Data from Mapper
+
+    # Long TE HERCULES Metabolite Data
+    lTE_metab        = copy.deepcopy(raw_data)                                              # Long TE Metab
+    lTE_mask         = np.ones(lTE_metab.shape[4], dtype=bool)                              # Create a Mask
+    lTE_mask[::33]   = False                                                                # Remove Water Refs
+    lTE_mask[: 33]   = False                                                                # Remove PRESS
+    lTE_metab        = lTE_metab[:, :, :, :, lTE_mask, :]                                   # Isolated HERCULES
+
+    # Handle Incomplete
+    if lTE_mask.shape[-1] % 4 != 0:                                                         # Incomplete Acquisition
+        old_num_avgs = lTE_mask.shape[-1]                                                   # Old Total Averages
+        new_num_avgs = (lTE_mask.shape[-1] // 4) * 4                                        # New Total Averages
+        lTE_metab    = lTE_metab[:, :, :, :, :new_num_avgs, :]                              # Remove Incomplete
+
+        notestring   = f'{subseq:3d} {subseq_name:<20}'                                     # Note Incomplete Data
```

Comment:
Hi @agudmundson There's these undefined variables `subseq` and `subseq_name`.  What should they be defined as?

Reply:
Hi @wtclarke! Ahh yes, that was a bit of code copied over to mirror the Twix implementation. Looks like you updated the string to 80ms HERCULES which is perfect! Did you get everything fixed or or is there anything else I should take a look at?

Reply:
Yes, in this regard, but now a bunch of errors occurring in the tests. I'm very certain that this is unrelated to your changes, and instead related to new package versions. I'll investigate.